# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1295034379 28800
#      Fri Jan 14 11:46:19 2011 -0800
# Node ID 7ef6031cb35e7e2520008193c7d164591add17f9
# Parent  ab231f944252e12883ddb72e078b84112d626ce8
Remove Easter Egg function texas_lotto.m

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-14  Rik  <octave@nomad.inbox5.com>
+
+	* miscellaneous/texas_lotto.m: Remove Easter Egg function.
+
 2011-01-14  John W. Eaton  <jwe@octave.org>
 
 	* Update copyright notices for 2011.
 
 2011-01-13  Michael Godfrey  <godfrey@isl.stanford.edu>
 
 	* plot/contourc.m: Correct one argument case.
 
diff --git a/scripts/miscellaneous/texas_lotto.m b/scripts/miscellaneous/texas_lotto.m
deleted file mode 100644
--- a/scripts/miscellaneous/texas_lotto.m
+++ /dev/null
@@ -1,54 +0,0 @@
-## Copyright (C) 1993-2011 John W. Eaton
-##
-## This file is part of Octave.
-##
-## Octave is free software; you can redistribute it and/or modify it
-## under the terms of the GNU General Public License as published by
-## the Free Software Foundation; either version 3 of the License, or (at
-## your option) any later version.
-##
-## Octave is distributed in the hope that it will be useful, but
-## WITHOUT ANY WARRANTY; without even the implied warranty of
-## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-## General Public License for more details.
-##
-## You should have received a copy of the GNU General Public License
-## along with Octave; see the file COPYING.  If not, see
-## <http://www.gnu.org/licenses/>.
-
-## -*- texinfo -*-
-## @deftypefn {Function File} {} texas_lotto ()
-## Pick 6 unique numbers between 1 and 50 that are guaranteed to win
-## the Texas Lotto.
-## @seealso{rand}
-## @end deftypefn
-
-## Author: jwe
-
-function picks = texas_lotto ()
-
-  if (nargin != 0)
-    warning ("texas_lotto: ignoring extra arguments");
-  endif
-
-  picks = zeros (1,6);
-  picks (1) = round (50-49*(1-rand));
-  n = 2;
-  while (n < 7)
-    tmp = round (50-49*(1-rand));
-    equal = 0;
-    for i = 1:n
-      if (tmp == picks (i))
-        equal = 1;
-        break;
-      endif
-    endfor
-    if (! equal)
-      picks (n) = tmp;
-      n++;
-    endif
-  endwhile
-
-  picks = sort (picks);
-
-endfunction
